go.sum

```diff
@@ -0,0 +1,2 @@
+golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e h1:+WEEuIdZHnUeJJmEUjyYC2gfUMj69yZXw17EnHg/otA=
```

Comment:
This file should be empty since we're not using any external package. Please run `go mod tidy` to fix this.